docs/src/content/docs/challenges/angular/45-simple-animations.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 🟢 Simple Animations
+description: Challenge 45 is about learning angular's integrated animation API
+author: sven-brodny
+challengeNumber: 45
+command: angular-simple-animations
+sidebar:
+  order: 17
+  badge: New
+---
+
+## Information
+
+Well-designed animations can make your application more fun and straightforward to use, but they aren't just cosmetic. Animations can improve your application and user experience in a number of ways:
+
+- Without animations, web page transitions can seem abrupt and jarring
+- Motion greatly enhances the user experience, so animations give users a chance to detect the application's response to their actions
+- Good animations intuitively call the user's attention to where it is needed
+
+I would recommend you to read the [official documentation](https://angular.io/guide/animations). You will learn everything that is necessary to successfully complete the challenge.
```

Comment:
remove `to`

Reply:
isn't that correct? you mean the first "to" in this? (... recommend you **to** read the ...)

Reply:
`I would recommend you read` is correct and flows better.  `to` is unnecessary.  Sorry, should I clarified better.

Reply:
done thx for your feedback!